src/Compilers/CSharp/Portable/Syntax/ArgumentSyntax.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel;
+
+namespace Microsoft.CodeAnalysis.CSharp.Syntax
+{
+    public sealed partial class ArgumentSyntax
+    {
+        /// <summary>
+        /// Pre C# 7.2 back-compat overload, which simply calls the replacement property <see cref="RefKindKeyword"/>.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Should these also have an Obsolete attribute, letting the user know they can update their code? `EditorBrowsableState.Never` hides the member from IntelliSense, but it doesn't clarify that the member has been replaced in other tools.

Reply:
I'd rather not obsolete APIs that are not actively harmful.